doc/release-process.md

```diff
@@ -291,24 +291,32 @@ bitcoin.org (see below for bitcoin.org update instructions).
   - After the pull request is merged, the website will automatically show the newest version within 15 minutes, as well
     as update the OS download links. Ping @saivann/@harding (saivann/harding on Freenode) in case anything goes wrong
 
-- Announce the release:
-
-  - bitcoin-dev and bitcoin-core-dev mailing list
-
-  - Bitcoin Core announcements list https://bitcoincore.org/en/list/announcements/join/
+- Update other repositories and websites for new version
 
   - bitcoincore.org blog post
 
   - bitcoincore.org RPC documentation update
 
-  - Update title of #bitcoin on Freenode IRC
+  - Update packaging repo
 
-  - Optionally twitter, reddit /r/Bitcoin, ... but this will usually sort out itself
+      - Notify BlueMatt so that he can start building [the PPAs](https://launchpad.net/~bitcoin/+archive/ubuntu/bitcoin)
+
+      - Create a new branch for the major release "0.xx" (used to build the snap package)
```

Comment:
Huh? We already have the branch by this point.

Reply:
This is for the packaging repo under bitcoin-core/packaging